README.md

```diff
@@ -1,31 +1,70 @@
 [RethinkDB](http://www.rethinkdb.com)
 =================
 
-RethinkDB is an open-source distributed database. It has an intuitive
-query language, automatically parallelized queries, and simple
-administration.
+RethinkDB is the first open-source scalable database built for realtime applications. It exposes a new database access model -- instead of polling for changes, the developer can tell the database to continuously push updated query results to applications in realtime. RethinkDB allows developers to build scalable realtime apps in a fraction of the time with less effort.
 
-To get started, check out [rethinkdb.com](http://rethinkdb.com)!
+To learn more, check out [rethinkdb.com](http://rethinkdb.com).
 
-Quick start
+Not sure what types of projects RethinkDB can help you build? Here are a few examples:
+
+* [Build a realtime liveblog with RethinkDB and Pubnub](http://rethinkdb.com/blog/rethinkdb-pubnub/)
+* [Create a collaborative photo sharing whiteboard](https://www.youtube.com/watch?v=pdPRp3UxL_s)
+* [Build an IRC bot in Go with RethinkDB changefeeds](http://rethinkdb.com/blog/go-irc-bot/)
+* [Look at cats on Instagram in realtime](http://rethinkdb.com/blog/cats-of-instagram/)
+* [Watch how Platzi uses RethinkDB to educate](https://www.youtube.com/watch?v=Nb_UzRYDB40)
+
+
+Quickstart
 -----------
 
-For a thirty second quickstart, look at http://www.rethinkdb.com/docs/guides/quickstart
+For a thirty-second RethinkDB quickstart, check out  [rethinkdb.com/docs/quickstart](http://www.rethinkdb.com/docs/quickstart).
+
+Or get started right away, with one of our language specific guides:
+
+* [Ten-minute guide with RethinkDB and Python](http://rethinkdb.com/docs/guide/python/)
+* [Ten-minute guide with RethinkDB and Ruby](http://rethinkdb.com/docs/guide/ruby/)
+* [Ten-minute guide with RethinkDB and JavaScript](http://rethinkdb.com/docs/guide/javascript/)
 
-Get help
+Besides our three official drivers, we also have many [community drivers](http://rethinkdb.com/docs/install-drivers/) supported by the RethinkDB community. Here's a few:
+
+* [C#/.NET rethinkdb-net](https://github.com/mfenniak/rethinkdb-net) - [@mfenniak](https://github.com/mfenniak)
```

Comment:
Could we use this formatting instead?

- __C# / .NET:__ [rethinkdb-net]()
- __Clojure:__ [GoRethink]()

Some projects have multiple authors, so in the interest of space, I'd just link to the projects.